inc/namespace.php

```diff
@@ -69,6 +69,18 @@ function bootstrap() {
 
 	// Disable the admin_email verification interval.
 	add_filter( 'admin_email_check_interval', '__return_zero' );
+
+	/*
+	 * Performance enhancements for comments as best practise.
```

Comment:
```suggestion
	 * Performance enhancements for comments as best practice.
```

Reply:
Doh, and I did google it... just missed that `practise` is a verb! Thanks.